packages/flutter/lib/src/material/search.dart

```diff
@@ -67,8 +67,8 @@ Future<T?> showSearch<T>({
 ///
 /// The search page always shows an [AppBar] at the top where users can
 /// enter their search queries. The buttons shown before and after the search
-/// query text field can be customized via [SearchDelegate.buildLeading] and
-/// [SearchDelegate.buildActions].
+/// query text field can be customized via [SearchDelegate.buildLeading]
+/// [SearchDelegate.buildBottom] and [SearchDelegate.buildActions].
```

Comment:
This sentence is referring to buttons before/after the search field. I think the `buildBottom` portion should be its own separate sentence.

Reply:
Updated!